astropy/units/tests/test_quantity.py

```diff
@@ -667,6 +667,13 @@ def test_quantity_comparison(self):
 
         assert u.Quantity(1200, unit=u.meter) != u.Quantity(1, unit=u.kilometer)
 
+    def test_quantity_is_equivalent(self):
```

Comment:
What about composite units? Say, `erg / s / cm^2 / Hz^2`? Is that considered equivalent to `Jy`?